templates/robotics/index.html

```diff
@@ -78,6 +96,33 @@ <h2 class="p-logo-section__title u-align--center">Proven in the field - companie
             ) | safe
           }}
         </div>
+
+        <div class="p-logo-section__item">
```

Comment:
These logos should be produced by design @lyubomir-popov

Reply:
Aren't the ones we are using designed by a designer? The come from assets server